# Exercício 01 — Classificação de carga em cabos de aço
**Estudante:** Mateus Carvalho


Este programa realiza a análise estrutural de um cabo de aço sob carga.
A partir do diâmetro e da força aplicada, o código calcula a área transversal, a tensão mecânica gerada e o fator de utilização do material.
Ao final, o sistema classifica o nível de risco operacional do cabo, auxiliando na tomada de decisão da engenharia.


In [ ]:
# Declaração das variáveis fornecidas pelo enunciado
diametro_mm = 12          # Diâmetro do cabo em milímetros (mm)
carga_N = 8500            # Força aplicada no cabo em Newtons (N)
tensao_ruptura_MPa = 400  # Tensão de ruptura do aço em Megapascals (MPa)


In [ ]:
import math

# 1. Conversão do diâmetro de milímetros para metros
diametro_m = diametro_mm / 1000

# 2. Cálculo da área da seção transversal circular: A = π * (d / 2)²
area_m2 = math.pi * (diametro_m / 2) ** 2

# 3. Cálculo da tensão aplicada em Pascal (Pa): σ = F / A
tensao_Pa = carga_N / area_m2

# 4. Conversão da tensão de Pascal (Pa) para Megapascal (MPa)
tensao_MPa = tensao_Pa / 1000000

# Exibição dos resultados intermediários exigidos
print(f"Área da seção calculada: {area_m2:.6f} m²")
print(f"Tensão aplicada calculada: {tensao_MPa:.2f} MPa")


Área da seção calculada: 0.000113 m²
Tensão aplicada calculada: 75.16 MPa


In [ ]:
# Cálculo do fator de utilização (razão entre a tensão atual e a de ruptura)
fator_utilizacao = tensao_MPa / tensao_ruptura_MPa

# Estrutura condicional para classificação do risco com base no fator
if fator_utilizacao < 0.50:
    classificacao = "Operação segura"
elif fator_utilizacao <= 0.74:
    classificacao = "Atenção — verificar periodicamente"
elif fator_utilizacao <= 0.89:
    classificacao = "Risco elevado — substituição recomendada"
else:
    classificacao = "Perigo — cabo fora de operação"


In [ ]:
# Relatório formatado utilizando f-strings com duas casas decimais
print("=== RELATÓRIO DE ANÁLISE DE CABO ===")
print(f"Diâmetro:            {diametro_mm:.2f} mm")
print(f"Carga aplicada:      {carga_N:.2f} N")
print(f"Área da seção:       {area_m2:.6f} m²")
print(f"Tensão aplicada:     {tensao_MPa:.2f} MPa")
print(f"Fator de utilização: {fator_utilizacao:.2f}")
print(f"Classificação:       {classificacao}")
print("=====================================")


=== RELATÓRIO DE ANÁLISE DE CABO ===
Diâmetro:            12.00 mm
Carga aplicada:      8500.00 N
Área da seção:       0.000113 m²
Tensão aplicada:     75.16 MPa
Fator de utilização: 0.19
Classificação:       Operação segura


### Interpretação do Resultado
O cabo de aço analisado encontra-se em **condição segura de operação**. O fator de utilização calculado foi de **0,19**, o que significa que o cabo está submetido a apenas 19% da sua capacidade máxima de ruptura (bem abaixo do limite de alerta de 50%). Como recomendação ao engenheiro responsável, a operação pode prosseguir normalmente, devendo apenas manter o cronograma padrão de inspeções visuais preventivas para desgaste e lubrificação.
